site/content/user-guide/amqp-adapter.md

```diff
@@ -51,13 +64,13 @@ To run the client using the above default values, open a terminal and execute th
 
     $ /hono/cli/target$ java -jar hono-cli-0.7-SNAPSHOT-exec.jar --spring.profiles.active=amqp-adapter-cli
 
-To run the client with a different value for the message address, username, password and payload body, we supply those values to command by doing the following:
+To run the client with a different value for the message address, username, password and payload body, we supply those values to the command by doing the following:
 
     $ /hono/cli/target$ java -jar hono-cli-0.7-SNAPSHOT-exec.jar --spring.profiles.active=amqp-adapter-cli --username=sensor20@DEFAULT_TENANT --password=my-secret --message.address=event/DEFAULT_TENANT/4710 --payload='{"alarm": 1}'
     
     Rejected{error=Error{condition=amqp:precondition-failed, description='device unknown or disabled', info=null}}
 
-After running the client, the delivery state is printed to standard output. As shown above, if we try to publish a message for a device that is not registered, the message will be rejected by the AMQP adapter by sending back the *rejected* disposition frame. The disposition frame received from the client is logged to standard output.
+After running the client, the delivery state is printed to standard output. As shown above, if we try to publish a message for a device that is not registered, the message will be rejected by the AMQP adapter by sending back the *rejected* disposition frame. The disposition frame received from the adapter is logged to standard output.
```

Comment:
no first person
before illustrating the failure to publish, it would be nice to see the output of a successful attempt to publish, don't you think?

Reply:
ok